### ISSUE TITLE
Improve description of restrictions for seed-phrase word count.

### ISSUE THREAD

**Issue by CypherPoet** [valeralabs/vcli#2]
This is the current help for the `length` option of the `new-seed-phrase` command: 

```
"total words in seed phrase, between 12 - 24."
```

I feel like this is somewhat incomplete, though. Even within that range, attempting to use anything other than 12, 15, 18, 21, or 24 will through an error from `VDK` stating: `Entropy length must be [128, 256] and a multiple of 32`.

This is correct, and relates to the following criteria:

- 128 bits of entropy maps to a 12 word seed phrase
- 160 bits of entropy maps to a 15 word seed phrase
- 192 bits of entropy maps to an 18 word seed phrase
- 224 bits of entropy maps to a 21 word seed phrase
- 256 bits of entropy maps to a 24 word seed phrase

It would be useful to reflect that in the aforementioned help text for the option -- and it would be trivial enough for `vcli` to validate this up front before even trying to call in to `VDKNewPhrase`.

**Comment by CypherPoet** [comment]
In fact, even better, I'm thinking we can make use of [Swift Argument Parser's `EnumerableFlag` type](https://github.com/apple/swift-argument-parser/blob/main/Sources/ArgumentParser/Parsable%20Types/EnumerableFlag.swift).

This would make the options self-describing -- and remove the error-prone nature of manually specifying a number.

**Comment by linden** [comment]
agreed we should clarify the error message for this but using an enumerable for a numerical input feels kind of strange.

I think it would make more sense to have `--words=12` just validated with a more helpful error message (on the `VDK`-side) and clarification of the possible lengths added to the `--help` menu.
